provider/resources.go

```diff
@@ -346,7 +346,11 @@ func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) erro
 
 	if _, err := awsbase.GetAwsConfig(context.Background(), config); err != nil {
 		return fmt.Errorf("unable to validate AWS credentials " +
-			"- see https://pulumi.io/install/aws.html for details on configuration")
+			"- see https://pulumi.io/install/aws.html for details on configuration\n" +
+			"log in to AWS using the AWS CLI:\n" +
+			"    aws configure\n" +
+			"log in to AWS using the AWS CLI(with SSO):\n" +
```

Comment:
capitalize, same as above

Reply:
maybe I could add a `$` to make it more clear that it is a shell command